src/git.rs

```diff
@@ -129,31 +131,32 @@ pub struct Repository {
 
     ref_map: RefMap,
     head: Head,
+    commit_hashes: Vec<CommitHash>,
 }
 
 impl Repository {
-    pub fn load(path: &Path) -> Self {
+    pub fn load(path: &Path, sort: SortCommit) -> Self {
```

Comment:
Could you please add a TODO comment to handle stashes?

Reply:
Done in https://github.com/lusingander/serie/pull/5/commits/74ab3d91509a19931ffc092d07ea2ba01fa5bdab